README.md

```diff
@@ -153,6 +153,55 @@ You can edit `camera.txt` on-target by remounting `/boot` read-write:
 mount -o remount,rw /boot
 ```
 
+### Configure available camera resolutions and streaming format
+
+The camera advertizes to the connected host PC the available resolutions and
+streaming formats of the video signal. Usually the best suitable resolution is
+chosen. The available resolutions can be reduced to enforce a specific setting.
+
+For example: MS Teams on Windows 10 is picking only a very low resolution and
+the picture looks blurry. To fix this, the low resolutions can be removed from
+the advertized set.
```

Comment:
Just a minor nitpick: Many haven't had this issue with Teams on WIndows, some did however. I'd guess this paragraph would be better suited for the FAQ (https://github.com/showmewebcam/showmewebcam/wiki/FAQ) than the Readme.

Reply:
I have moved the use case from the README to the wiki.